packages/calcite-components/src/components/checkbox/checkbox.scss

```diff
@@ -63,7 +72,7 @@
 :host([hovered]) .toggle,
 :host .toggle:hover {
   .check-svg {
-    box-shadow: inset 0 0 0 2px var(--calcite-color-brand);
+    box-shadow: inset 0 0 0 2px var(--calcite-checkbox-border-color-hover, var(--calcite-color-brand));
```

Comment:
I think we can simplify this. There is no reason to have a -hover token here if we simplify the selectors.

```css
:host([hovered]) .toggle,
:host(:hover) .toggle {
```

Reply:
I’m confused about when I should expect a stateful prop versus a non-stateful one. This seems to contradict the [recent CSS props added to the action family](https://github.com/Esri/calcite-design-system/pull/10058/) (e.g., `--calcite-action-background-color-hover`).

Am I missing something? There’s no mention or example of this in the Component Tokens wiki.

This was also brought up earlier ([see discussion](https://github.com/Esri/calcite-design-system/pull/10221#discussion_r1752671869)).

Could you clarify?

Reply:
Sorry. I'm mistaken. Yeah I'm sorry. Per https://github.com/Esri/calcite-design-system/wiki/component-tokens#naming-pattern we're using -hover here